spec/shr_oncologycore.txt

```diff
@@ -42,151 +42,167 @@ Concept:        LNC#21858-6 "Grade Cancer"
                 FindingResult from http://loinc.org/vs/LL240-3
-                FindingTopicCode is MTH#C1302362 "Stage Grouping" 
+                FindingTopicCode is MTH#C1302362 "Stage Grouping"
+0..1            FindingMethod from TNMTumorStagingSystemVS if covered
+                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
+1..1            CancerStageTiming
+0..0			AnatomicalLocation
+0..0            NonIndependentFinding
+0..0            PanelMembers
+0..0            Device
+
+        EntryElement:   TNMClinicalStageGroup
+        Based on:       TNMStageGroup
+        Concept:        TBD
+        Description:    "TBD"
+                        CancerStageTiming is SCT#385349001   "Clinical stage (observable entity)"
 
+        EntryElement:   TNMPathologicStageGroup
+        Based on:       TNMStageGroup
+        Concept:        TBD
+        Description:    "TBD"
+                        CancerStageTiming is SCT#405978005   "Pathologic stage (observable entity)" 
+        
 Element:        StageSuffix
-Based on:       CodedNonIndependentFinding
 Concept:	    TBD#TBD	    // TODO: Needs a new LOINC or SNOMED code + value set terms.  assign a local value set until we get something assigned.
 Description: "A suffix used in conjuction with certain cancer stage categories, usually qualifying how the category value was determined."
-                FindingResult from StageSuffixVS  // local value set created 
-                FindingTopicCode is TBD#TBD  // TODO: Needs a new LOINC or SNOMED code + value set terms.
+Value:          CodeableConcept from StageSuffixVS  // local value set created 
+
+Abstract Element:   TNMClassification
+Based on:		    CodedNonLaboratoryObservation
+Concept:		    LNC#21905-5 "Primary tumor.clinical [Class] Cancer"
+Description:        "Abstract class parent of TNM stage classifications."
+                    SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
+1..1                CancerStageTiming
+0..1                FindingMethod from TNMTumorStagingSystemVS if covered
+0..1                StageSuffix
+0..0			    AnatomicalLocation
+0..0                NonIndependentFinding
+0..0                PanelMembers
+0..0                Device
+
+Abstract Element:   TNMClinicalClassification
+Based on:           TNMClassification
+Concept:            TBD
+Description:        "Abstract class parent for TNM clinical stage classifications."
+                    CancerStageTiming is SCT#385349001 "Clinical stage (observable entity)"
+
+Abstract Element:   TNMPathologicClassification
+Based on:           TNMClassification
+Concept:            TBD
+Description:        "Abstract class parent for TNM clinical stage classifications."
```

Comment:
"Abstract class parent for TNM **pathologic** stage classifications."

Reply:
Good catch.